system/otherLoadTest/playlist.xml

```diff
@@ -339,7 +339,10 @@
 		<disables>
 			<disable>
 				<comment>https://github.com/eclipse-openj9/openj9/issues/15250</comment>
-				<impl>openj9</impl>
+				<impls>
+					<impl>openj9</impl>
+					<impl>ibm</impl>				
+				</impls>
```

Comment:
I do not think `<impls>` is not supported in `<disable>`. Please add separate `<disable>` for `ibm`.
Please run Grinder for your PR. For details, please see Step 9-10 in https://github.com/adoptium/aqa-tests/blob/master/Contributing.md